.gitignore

```diff
@@ -4,5 +4,4 @@ coverage
 reports
 **/junit.xml
 .idea
-bin/**/*.js
-src/**/*.js
+dist
```

Comment:
@ChristopheBougere

I moved the JS output into a `dist` folder which is now part of .gitignore. This removes the need to filter js files from src since we shouldn't see them in the src folder.